main.py

```diff
@@ -0,0 +1,9 @@
+from scraper import get_page_urls, get_product_urls, get_all_product_details
+import pandas as pd
```

Comment:
import rules

1. built-in libraries
2. third party libraries
3. local imports